<?xml version="1.0"?>
<launch>

    <!-- launch-prefix="xterm -e gdb -ex run args" -->
    <node pkg="ros_examples" type="example_rrt_exploration_ogm" name="example_rrt_exploration_ogm" output="screen" respawn="true">
        <param name="debug" value="false"/>
        <!-- eta must be greater than map resolution -->
        <param name="eta" value="0.5"/>
        <param name="min_dist" value="0.25"/>
        <param name="marker_size" value="0.025"/>
        <param name="rate_hz" value="1000"/>
        <param name="map_topic" value="/map"/>
        <remap from="clicked_point" to="/clicked_point"/>
        <remap from="rrt" to="rrt"/>
        <remap from="frontier" to="frontier"/>
    </node>

    <node pkg="map_server" type="map_server" name="map_server" args="$(dirname)/../map/sample_map.yaml" output="screen">
        <remap from="map" to="/map"/>
        <param name="frame_id" value="map"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(dirname)/../rviz/example_rrt_ogm.rviz" required="true"/>

</launch>